src/main/webapp/dashboard.js

```diff
@@ -1,49 +1,92 @@
+/**
+ * A project
+ * @typedef {Object} Project
+ * @property {String} name - the name of a given project.
+ * @property {Mentee List} mentees - the list of mentees under a given project.
+ */
+
+/**
+ * A mentee
+ * @typedef {Object} Mentee
+ * @property {String} image -the link to a mentee's image
+ * @property {String} name - the name of the mentee
+ * @property {String} starLink - the link to the action that rewards a given
```

Comment:
Just a Q: What actions do you think may deserve stars in the MVP version?

Reply:
I don't know, I thought it would be at the discretion of the mentor. I might remove this and make it a stretch goal.